cmake_modules/optionDefaults.cmake

```diff
@@ -2,20 +2,21 @@ if ( NOT DESTDIR )
     set( DESTDIR "" )
 endif()
 
-if ( NOT PREFIX )
-if ( WIN32 )
-    set( prog86key "ProgramFiles(x86)" )
-    if ("$ENV{${prog86key}}" STREQUAL "")
-        set( PREFIX "$ENV{ProgramFiles}" )
-    else()
-        set( PREFIX "$ENV{${prog86key}}" )
-    endif()
-    set(ORIG_PREFIX ${PREFIX})
-    string(REGEX REPLACE "\\\\" "/" PREFIX ${PREFIX})    
-else (WIN32)
-    set( PREFIX "${DESTDIR}/opt" )
-    set( ORIG_PREFIX ${PREFIX})
-endif (WIN32)
+if(NOT PREFIX)
+    if(WIN32)
+        set(prog86key "ProgramFiles(x86)")
+        if("$ENV{${prog86key}}" STREQUAL "")
+            set(PREFIX "$ENV{ProgramFiles}")
+        else()
+            if("${CMAKE_GENERATOR}" STREQUAL "Visual Studio 14 2015")
```

Comment:
@GordonSmith Please take a quick look.  If we are on a 64 bit target and are building for 32 bit, we now appropriately set the prefix to ProgramFiles(x86), and ProgramFiles otherwise.

Reply:
Ok so this test:  ```if("${CMAKE_GENERATOR}" STREQUAL "Visual Studio 14 2015")``` is your way of checking if the current package format is 32bit (on the build machine)?

Reply:
Ok so this test:  ```if("${CMAKE_GENERATOR}" STREQUAL "Visual Studio 14 2015")``` is your way of checking if the current package target is 32bit?  If so it looks ok.